### PR TITLE
Add brigmedic role

### PR DESCRIPTION
**Pull request by fqqf**
## About the PR
image

adds brigmedic role

- requires 15 hours med and 20 hours security
- brigmedic will now roundstart have mk 58 and medsec hud 
- salary is 68

## Why / Balance
because tamashi mains brigmedic and i am tired of him (https://discord.com/channels/1272545509562777621/1311593820236025876)

## Media
https://github.com/user-attachments/assets/62ec00b5-0917-436c-889c-394d877656d5

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl: Miracula
- add: Added brigmedic role

### CODE REVIEW

**Review by Rinary1 — APPROVED** [review]

File: Resources/Prototypes/_StarLight/Roles/Jobs/Security/brigmedic.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    mask: ClothingMaskBreathMedicalSecurity
+    gloves: ClothingHandsGlovesNitrile
+  storage:
+    back:
+    - BoxSurvivalSecurity
+    - Flash
+    - MagazinePistol
+  innerClothingSkirt: ClothingUniformJumpsuitBrigmedic
+  satchel: ClothingBackpackBrigmedic
+  duffelbag: ClothingBackpackBrigmedic
```
Comment (Rinary1) [inline Resources/Prototypes/_StarLight/Roles/Jobs/Security/brigmedic.yml]: linter

**Review by Rinary1 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_StarLight/Roles/Jobs/Security/brigmedic.yml (hunk 1 above)
Comment (fqqf) [reply Resources/Prototypes/_StarLight/Roles/Jobs/Security/brigmedic.yml]: oof, my apologies. Left that to remind myself to do loadout. Will fix in few hours

**Comment by fqqf** [comment]
added loadout


https://github.com/user-attachments/assets/9ffac801-a5b4-464c-a510-e88975489979

**Comment by fqqf** [comment]
alright, everything seems good now

**Review by Rinary1 — APPROVED** [review]

**Pull request merged**